docs/INSTALLATION.md

```diff
@@ -0,0 +1,263 @@
+* [PM2](http://pm2.keymetrics.io/) (install with the command: `npm install -global pm2`)
+* [Lerna](https://lernajs.io/) (install with the command: `npm install -global lerna`)
+
+Compiling source code for the Docker runtime also requires:
+
+* [Docker](https://www.docker.com/get-started)
+* [Docker Compose](https://docs.docker.com/compose/install/)
+
+Building [**OVE Asset Services**](https://github.com/ove/ove-asset-services) also requires:
+
+* [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)
+* [NetVips](https://github.com/kleisauke/net-vips)
+* [MariaDB](https://mariadb.org/)
+
+The [SVG App](https://github.com/ove/ove-apps/tree/master/packages/ove-app-svg) requires:
+
+* [Tuoris](https://github.com/fvictor/tuoris) (installation instructions available on GitHub repository)
+
+### Downloading source code
+
+All OVE projects can be downloaded from their GitHub repositories:
+
+* OVE Core: [master](https://github.com/ove/ove) | [releases](https://github.com/ove/ove/releases)
+* OVE Apps: [master](https://github.com/ove/ove-apps) | [releases](https://github.com/ove/ove-apps/releases)
+* OVE Services: [master](https://github.com/ove/ove-services) | [releases](https://github.com/ove/ove-services/releases)
+* OVE Asset Services: [master](https://github.com/ove/ove-asset-services) | [releases](https://github.com/ove/ove-asset-services/releases)
+
+The `master` branch of each repository contains the latest code, and can also be cloned if you intend to contribute code or fix issues:
+
+```sh
+git clone https://github.com/ove/ove
+```
+
+Once the source code has been downloaded OVE can be installed either onto a PM2 runtime or a Docker runtime. The two approaches are explained below.
+
+### Compiling source code for the PM2 runtime using Lerna
+
+Once you have cloned or downloaded the code OVE can be compiled using the [Lerna](https://lernajs.io/) build system:
+
+```sh
+cd ove
+lerna bootstrap --hoist
+lerna run clean
+lerna run build
+lerna run test
+```
+
+Instructions above are only provided for the [**OVE Core**](https://github.com/ove/ove) repository. The steps to follow are similar for other repositories.
+
+#### Starting and stopping OVE
+
+The SVG app requires an instance of [Tuoris](https://github.com/fvictor/tuoris) to be available before starting it:
+
+```sh
+pm2 start index.js -f -n "tuoris" -- -p PORT -i 1
+```
+
+To start the PM2 environment run:
+
+```sh
+OVE_HOST="OVE_CORE_HOST:PORT" TUORIS_HOST="TUORIS_HOST:PORT" pm2 start pm2.json
+```
+
+Once the services have started, you can check their status by running:
+
+```sh
+pm2 status
+```
+
+Then, to check logs of all services, run:
+
+```sh
+pm2 logs
+```
+
+To stop the PM2 environment run:
+
+```sh
+pm2 stop pm2.json
+```
+
+To clean-up the PM2 environment run:
+
+```sh
+pm2 delete pm2.json
+```
+
+### Compiling source code for the Docker runtime using build.sh scripts
```

Comment:
```suggestion
### Compiling source code inside Docker using build.sh scripts
```

Reply:
As also explained earlier, this does not run entirely inside docker.